department-of-veterans-affairs/caseflow
# APPEALS-24881

Resolves #{24881}
[APPEALS-24881](https://jira.devops.va.gov/browse/APPEALS-24881)

# Description
This bug fix corrects the order that information is displayed in the case timeline section.

## Acceptance Criteria

## Testing Plan

1. Go to #{24881} [APPEALS-24881](https://jira.devops.va.gov/browse/APPEALS-24881)
2. Check out the latest code
3. Log in as Judge (BVAAABSHIRE)
4. Proceed to Queue
5. Select an AMA or LEGACY Appeal (Both need to be tested and have their own output to be confirmed).
6. Under Actions, activate dropdown and select "Ready for Dispatch"
7. Select Add Decision (Edit Issue for Legacy Appeal)
8. Make a change to the MST / PACT checkbox selection
9. Select Continue and complete the form.
10. Search for the Veteran ID and go to the case timeline section.
11. Ensure that the items are displayed in the proper order listed if it is an AMA or LEGACY Appeal.

Also check that the correct issue workflow is outputting the same data.
1. Log in as BVADWISE
2. Select an Appeal and visit its case details page
3. Select the "Correct Issues" link
4. Select Edit Issue
5. Change the MST / PACT status on the issue.
6. Select Continue.
7. Ensure that the items are displayed in the proper order in the case timeline section.

Expected Result:
Note attached images for reference, information should be displayed if available.
1. For AMA the following should be in order: (1: Issue, 2: Benefit Type)
8. For LEGACY the following should be in order: (1: Benefit Type, 2: Issue, 3: Code, 4: Note, 5: Disposition, 6: Special Issues)

# Frontend
## User Facing Changes

![legacy issue order](https://github.com/department-of-veterans-affairs/caseflow/assets/33578594/672adb9d-56d2-4ca8-a36b-601b0bcbafa1)
![Legacy Expected](https://github.com/department-of-veterans-affairs/caseflow/assets/33578594/ea2c4cbe-b6e0-4be4-9613-966ee8d21624)
<img width="296" alt="Screenshot 2023-07-06 at 2 02 33 PM" src="https://github.com/department-of-veterans-affairs/caseflow/assets/33578594/349d3b6f-651d-46b2-9468-d9152000095d">
<img width="298" alt="Screenshot 2023-07-06 at 2 07 26 PM" src="https://github.com/department-of-veterans-affairs/caseflow/assets/33578594/c15484aa-b52b-4f8c-b722-5fd05bc8a47f">
